examples/tests/valid/api.w

```diff
@@ -13,7 +13,10 @@ let handler = inflight (request: cloud.ApiRequest): cloud.ApiResponse => {
   };
   
   let resp = cloud.ApiResponse {
-    body: bodyResponse,
+    body: Json.stringify(bodyResponse),
+    headers: {
+      "content-type": "application/json"
+    },
```

Comment:
Probably a follow up, but couldn't there be something like `cloud.ApiJsonResponse` to cover the default json cases as before and have the generic `cloud.ApiResponse` to let the user fully control the response?

Reply:
Yes. We need to add some sugar APIs on top.